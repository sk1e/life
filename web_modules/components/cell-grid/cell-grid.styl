json('./constants.json')

.cell-grid
  margin-top cell-grid-margin-top
