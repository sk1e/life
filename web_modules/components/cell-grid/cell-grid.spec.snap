exports[`CellGrid renders correctly`] = `
Object {
  "$$typeof": Symbol(react.element),
  "_owner": null,
  "_store": Object {},
  "key": null,
  "props": Object {
    "children": Array [
      " ",
      Array [
        Object {
          "$$typeof": Symbol(react.element),
          "_owner": null,
          "_store": Object {},
          "key": "0",
          "props": Object {
            "children": Array [
              " ",
              Array [
                Object {
                  "$$typeof": Symbol(react.element),
                  "_owner": null,
                  "_store": Object {},
                  "key": "0",
                  "props": Object {
                    "column": 0,
                    "row": 0,
                    "style": Object {
                      "height": "calc(100vw / 4 - 0.5rem)",
                      "maxHeight": "calc(100vh / 3 - 2rem)",
                      "maxWidth": "calc(100vh / 3 - 2rem)",
                      "width": "calc(100vw / 4 - 0.5rem)"
                    }
                  },
                  "ref": null,
                  "type": [Function Connect]
                },
                Object {
                  "$$typeof": Symbol(react.element),
                  "_owner": null,
                  "_store": Object {},
                  "key": "1",
                  "props": Object {
                    "column": 1,
                    "row": 0,
                    "style": Object {
                      "height": "calc(100vw / 4 - 0.5rem)",
                      "maxHeight": "calc(100vh / 3 - 2rem)",
                      "maxWidth": "calc(100vh / 3 - 2rem)",
                      "width": "calc(100vw / 4 - 0.5rem)"
                    }
                  },
                  "ref": null,
                  "type": [Function Connect]
                },
                Object {
                  "$$typeof": Symbol(react.element),
                  "_owner": null,
                  "_store": Object {},
                  "key": "2",
                  "props": Object {
                    "column": 2,
                    "row": 0,
                    "style": Object {
                      "height": "calc(100vw / 4 - 0.5rem)",
                      "maxHeight": "calc(100vh / 3 - 2rem)",
                      "maxWidth": "calc(100vh / 3 - 2rem)",
                      "width": "calc(100vw / 4 - 0.5rem)"
                    }
                  },
                  "ref": null,
                  "type": [Function Connect]
                },
                Object {
                  "$$typeof": Symbol(react.element),
                  "_owner": null,
                  "_store": Object {},
                  "key": "3",
                  "props": Object {
                    "column": 3,
                    "row": 0,
                    "style": Object {
                      "height": "calc(100vw / 4 - 0.5rem)",
                      "maxHeight": "calc(100vh / 3 - 2rem)",
                      "maxWidth": "calc(100vh / 3 - 2rem)",
                      "width": "calc(100vw / 4 - 0.5rem)"
                    }
                  },
                  "ref": null,
                  "type": [Function Connect]
                }
              ]
            ],
            "className": "cell-grid__row"
          },
          "ref": null,
          "type": "div"
        },
        Object {
          "$$typeof": Symbol(react.element),
          "_owner": null,
          "_store": Object {},
          "key": "1",
          "props": Object {
            "children": Array [
              " ",
              Array [
                Object {
                  "$$typeof": Symbol(react.element),
                  "_owner": null,
                  "_store": Object {},
                  "key": "0",
                  "props": Object {
                    "column": 0,
                    "row": 1,
                    "style": Object {
                      "height": "calc(100vw / 4 - 0.5rem)",
                      "maxHeight": "calc(100vh / 3 - 2rem)",
                      "maxWidth": "calc(100vh / 3 - 2rem)",
                      "width": "calc(100vw / 4 - 0.5rem)"
                    }
                  },
                  "ref": null,
                  "type": [Function Connect]
                },
                Object {
                  "$$typeof": Symbol(react.element),
                  "_owner": null,
                  "_store": Object {},
                  "key": "1",
                  "props": Object {
                    "column": 1,
                    "row": 1,
                    "style": Object {
                      "height": "calc(100vw / 4 - 0.5rem)",
                      "maxHeight": "calc(100vh / 3 - 2rem)",
                      "maxWidth": "calc(100vh / 3 - 2rem)",
                      "width": "calc(100vw / 4 - 0.5rem)"
                    }
                  },
                  "ref": null,
                  "type": [Function Connect]
                },
                Object {
                  "$$typeof": Symbol(react.element),
                  "_owner": null,
                  "_store": Object {},
                  "key": "2",
                  "props": Object {
                    "column": 2,
                    "row": 1,
                    "style": Object {
                      "height": "calc(100vw / 4 - 0.5rem)",
                      "maxHeight": "calc(100vh / 3 - 2rem)",
                      "maxWidth": "calc(100vh / 3 - 2rem)",
                      "width": "calc(100vw / 4 - 0.5rem)"
                    }
                  },
                  "ref": null,
                  "type": [Function Connect]
                },
                Object {
                  "$$typeof": Symbol(react.element),
                  "_owner": null,
                  "_store": Object {},
                  "key": "3",
                  "props": Object {
                    "column": 3,
                    "row": 1,
                    "style": Object {
                      "height": "calc(100vw / 4 - 0.5rem)",
                      "maxHeight": "calc(100vh / 3 - 2rem)",
                      "maxWidth": "calc(100vh / 3 - 2rem)",
                      "width": "calc(100vw / 4 - 0.5rem)"
                    }
                  },
                  "ref": null,
                  "type": [Function Connect]
                }
              ]
            ],
            "className": "cell-grid__row"
          },
          "ref": null,
          "type": "div"
        },
        Object {
          "$$typeof": Symbol(react.element),
          "_owner": null,
          "_store": Object {},
          "key": "2",
          "props": Object {
            "children": Array [
              " ",
              Array [
                Object {
                  "$$typeof": Symbol(react.element),
                  "_owner": null,
                  "_store": Object {},
                  "key": "0",
                  "props": Object {
                    "column": 0,
                    "row": 2,
                    "style": Object {
                      "height": "calc(100vw / 4 - 0.5rem)",
                      "maxHeight": "calc(100vh / 3 - 2rem)",
                      "maxWidth": "calc(100vh / 3 - 2rem)",
                      "width": "calc(100vw / 4 - 0.5rem)"
                    }
                  },
                  "ref": null,
                  "type": [Function Connect]
                },
                Object {
                  "$$typeof": Symbol(react.element),
                  "_owner": null,
                  "_store": Object {},
                  "key": "1",
                  "props": Object {
                    "column": 1,
                    "row": 2,
                    "style": Object {
                      "height": "calc(100vw / 4 - 0.5rem)",
                      "maxHeight": "calc(100vh / 3 - 2rem)",
                      "maxWidth": "calc(100vh / 3 - 2rem)",
                      "width": "calc(100vw / 4 - 0.5rem)"
                    }
                  },
                  "ref": null,
                  "type": [Function Connect]
                },
                Object {
                  "$$typeof": Symbol(react.element),
                  "_owner": null,
                  "_store": Object {},
                  "key": "2",
                  "props": Object {
                    "column": 2,
                    "row": 2,
                    "style": Object {
                      "height": "calc(100vw / 4 - 0.5rem)",
                      "maxHeight": "calc(100vh / 3 - 2rem)",
                      "maxWidth": "calc(100vh / 3 - 2rem)",
                      "width": "calc(100vw / 4 - 0.5rem)"
                    }
                  },
                  "ref": null,
                  "type": [Function Connect]
                },
                Object {
                  "$$typeof": Symbol(react.element),
                  "_owner": null,
                  "_store": Object {},
                  "key": "3",
                  "props": Object {
                    "column": 3,
                    "row": 2,
                    "style": Object {
                      "height": "calc(100vw / 4 - 0.5rem)",
                      "maxHeight": "calc(100vh / 3 - 2rem)",
                      "maxWidth": "calc(100vh / 3 - 2rem)",
                      "width": "calc(100vw / 4 - 0.5rem)"
                    }
                  },
                  "ref": null,
                  "type": [Function Connect]
                }
              ]
            ],
            "className": "cell-grid__row"
          },
          "ref": null,
          "type": "div"
        }
      ]
    ],
    "className": "cell-grid"
  },
  "ref": null,
  "type": "div"
}
`;
